commander3/src/comm_tod_gain_mod.f90

```diff
@@ -788,35 +836,73 @@ subroutine get_pid_ranges_tabulated(pid_ranges, tod)
 !!$     ! This last event is too close to the previous one
 !!$     pid_ranges(:, 16) = 0
 
-     pid_ranges(:, 1) = 1
-     pid_ranges(:, 2) = 3352
-     pid_ranges(:, 3) = 5030
-     pid_ranges(:, 4) = 5484
-     pid_ranges(:, 5) = 8309 ! 20K
-     pid_ranges(:, 6) = 8503 ! 20K
-     pid_ranges(:, 7) = 8606 ! 20K
-     pid_ranges(:, 8) = 9613 ! 20K
-     pid_ranges(:, 9) = 10117 ! 20K
-     pid_ranges(:, 10) = 10512 ! 20K
-     ! There's one more 20K at 10897 but that's very close to this one
-     pid_ranges(:, 11) = 10911
-     pid_ranges(:, 12) = 14061 ! 20K
-     pid_ranges(:, 13) = 15957
-     pid_ranges(:, 14) = 16204 ! 20K
-     pid_ranges(:, 15) = 16455
-     pid_ranges(:, 16) = 17024 ! 20K
-     pid_ranges(:, 17) = 18338 ! 20K
-     pid_ranges(:, 18) = 21484
-     pid_ranges(:, 19) = 25654
-     pid_ranges(:, 20) = 27110
-     pid_ranges(:, 21) = 27343
-     pid_ranges(:, 22) = 30387
-     pid_ranges(:, 23) = 32763
-     pid_ranges(:, 24) = 38591
-     pid_ranges(:, 25) = 43929
-     ! pid_ranges(:, 16) = 44063
-     ! This last event is too close to the previous one
-     pid_ranges(:, 26) = 0
+!     pid_ranges(:, 1) = 1
```

Comment:
Whatever this is, it's a trainwreck. This should 100% be an input file

Reply:
agreed

Reply:
Fix later.